ELY_Messages.txt

```diff
@@ -0,0 +1,120 @@
+# For the used ranges there may still be some gaps, the intent here has
+# been to try and collect closely related IDs into a range.
+
+00012            wildfly-elytron-asn1
+07004 - 07024    wildfly-elytron-asn1
+11001 - 11007    wildfly-elytron-audit
+00003            wildfly-elytron-auth-util
+01065            wildfly-elytron-auth-util
+01121 - 01123    wildfly-elytron-auth-util
+01160 - 01165    wildfly-elytron-auth-util
+03031            wildfly-elytron-auth-util
+00001            wildfly-elytron-base
+00005            wildfly-elytron-base
+00011            wildfly-elytron-base
+01106            wildfly-elytron-client
+05053            wildfly-elytron-client
+05125            wildfly-elytron-client
+09001            wildfly-elytron-client
+00002 - 00004    wildfly-elytron-credential
+01037            wildfly-elytron-credential
+01053 - 01063    wildfly-elytron-credential
+02032 - 02033    wildfly-elytron-credential
+05151            wildfly-elytron-credential
+08001 - 08029    wildfly-elytron-credential
+09000            wildfly-elytron-credential
+01030            wildfly-elytron-credential-source-deprecated
+05053            wildfly-elytron-credential-source-deprecated
+05125            wildfly-elytron-credential-source-deprecated
+09506            wildfly-elytron-credential-source-deprecated
+09511            wildfly-elytron-credential-source-deprecated
+01030            wildfly-elytron-credential-source-impl
+05053            wildfly-elytron-credential-source-impl
+05125            wildfly-elytron-credential-source-impl
+09506            wildfly-elytron-credential-source-impl
+09511            wildfly-elytron-credential-source-impl
+02009            wildfly-elytron-credential-store
+09500 - 09528    wildfly-elytron-credential-store
+11006            wildfly-elytron-credential-store
+06000 - 06018    wildfly-elytron-http
+06008 - 06014    wildfly-elytron-http-sso
+05172            wildfly-elytron-http-util
+03018            wildfly-elytron-jacc
+08500 - 08508    wildfly-elytron-jacc
+01167 - 01177    wildfly-elytron-jaspi
+02001 - 02035    wildfly-elytron-keystore
+08027            wildfly-elytron-keystore
+00007            wildfly-elytron-mechanism
+01151            wildfly-elytron-mechanism
+05001 - 05177    wildfly-elytron-mechanism
+06001 - 06022    wildfly-elytron-mechanism
+07001 - 07004    wildfly-elytron-mechanism
+00003            wildly-elytron-mechanism-gssapi
+01121 - 01123    wildly-elytron-mechanism-gssapi
+01160 - 01165    wildly-elytron-mechanism-gssapi
+03035            wildly-elytron-mechanism-gssapi
```

Comment:
The range in ```gssapi/src/main/java/org/wildfly/security/mechanism/gssapi/ElytronMessages.java``` is ```@ValidIdRange(min = 3031, max = 3031)```, which is correct. So looks like 3035 in the text file should be updated to 3031. This is just a minor change though, not a reason to hold up this PR.

Reply:
Created https://github.com/wildfly-security/wildfly-elytron/pull/1312 for that change.